coding-concepts.js

```diff
@@ -67,7 +67,10 @@ class Learn {
   }
 }
 const learnStudent = new Learn("George")
-// console.log(learnStudent)
+console.log(learnStudent)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: the console.log will apply the argument within the instanse into the class to change the first object this.student = name to this.student = "gearoge". 
+
+//this will bring back the entire class with the new argument as the value for the key this.student.
+
+// b) Verify and explain: answer was correct because the given variable is a class with objects and the class has a parameter of name within the constructor. there is also a variable with the instance of the class with a value of "george" (const learnStudent = new Learn("George") by logging learn students we initate the action to apply george into our class directly into the this.student = name. the reason it is placed specifically in that area is because the parameter is called name so whatever the parameter is it will direct the value into name.
```

Comment:
placing the parameter of name in the constructor is the signal that every time a new instance of the Learn class is created. It has to have one argument that will be displayed as the name parameter.